### PR TITLE
issue-112 Order taxi form landing page and fix logo icon and wording on a page

### PR DESCRIPTION
**Pull request by KurochkaR**
change static configurations, add static to template(index), add model for subscriber, fix validation for phone number, add forms(order and subscribe) to template

### CODE REVIEW

**Comment by SpaceIgor** [comment]
@KurochkaR 
1.Resolve conflicts
2.Return the file docker-compose_example.yml to the project
3.Align the inscription 
Снимок экрана 2023-02-24 в 12 50 20
4.Everything must be in english. In the example (email address wrong return)
Снимок экрана 2023-02-24 в 13 09 04
5. Add test(pytest) to your form

**Comment by KurochkaR** [comment]
app/models.py conflict in phone_validator i think couase i improve it for normal work, cause he made valid short numbers (ex. 8095123456)

**Comment by SpaceIgor** [comment]
> app/models.py conflict in phone_validator i think cause i improve it for normal work, he made valid short numbers (ex. 8095123456)
need +380.....

**Comment by KurochkaR** [comment]
image

File: Dockerfile [hunk 1]
```diff
@@ -6,4 +6,5 @@ COPY requirements.txt .
 RUN sudo pip install -r requirements.txt
 COPY . .
 RUN sudo chown nobody:nogroup /app
-EXPOSE 8080 44300
\ No newline at end of file
+EXPOSE 8080 44300
+CMD ["honcho", "start"]
```
Comment (SergeyKutsko) [inline Dockerfile:10]: remove

File: app/admin.py [hunk 2]
```diff
@@ -171,7 +171,7 @@ class DriverManagerAdmin(admin.ModelAdmin):
     list_per_page = 25
 
     fieldsets = [
-        (None, {'fields': ['name', 'second_name', 'email', 'phone_number', 'chat_id']}),
```
Comment (SergeyKutsko) [inline app/admin.py:174]: back chat_id

File: app/admin.py [hunk 3]
```diff
@@ -387,8 +387,4 @@ class OwnerAdmin(admin.ModelAdmin):
 
 @admin.register(Order)
 class OrderAdmin(admin.ModelAdmin):
-    list_display = [f.name for f in Order._meta.fields]
-
-@admin.register(Fleets_drivers_vehicles_rate)
```
Comment (SergeyKutsko) [inline app/admin.py:392]: why deleted?

File: app/models.py [hunk 4]
```diff
@@ -415,6 +421,10 @@ class Driver(User):
     OFFLINE = 'Не працюю'
    
     fleet = models.OneToOneField('Fleet', blank=True, null=True, on_delete=models.SET_NULL)
+    #driver_manager_id: ManyToManyField already exists in DriverManager
```
Comment (SergeyKutsko) [inline app/models.py:424]: WTF?

File: app/models.py [hunk 5]
```diff
@@ -649,11 +659,9 @@ def download_daily_report(self, day=None, driver=True, sleep=5, headless=True):
 
 
 class Vehicle(models.Model):
-    ELECTRO = 'Електро'
-
     name = models.CharField(max_length=255, verbose_name='Назва')
     model = models.CharField(max_length=50, verbose_name='Модель')
-    type = models.CharField(max_length=20, default=ELECTRO, verbose_name='Тип')
+    type = models.CharField(max_length=20, verbose_name='Тип')
```
Comment (SergeyKutsko) [inline app/models.py:664]: Back orignal

File: app/models.py [hunk 6]
```diff
@@ -1931,7 +1936,7 @@ def download_payments_order(self):
         self.get_target_page_or_login(url, xpath, self.login)
 
         self.driver.find_element(By.XPATH, '//flt-group-filter[1]/flt-date-range-filter/mat-form-field/div').click()
-        self.driver.find_element(By.XPATH, '//mat-option[@id="mat-option-7"]/span').click()  #Минулий тиждень
+        self.driver.find_element(By.XPATH, '//mat-option[@id="mat-option-8"]/span').click()  #Минулий тиждень
```
Comment (SergeyKutsko) [inline app/models.py:1939]: please back to original or make it better

File: app/tests/taxi_service_test/test_forms.py [hunk 7]
```diff
@@ -0,0 +1,46 @@
+#  -*- coding: utf-8 -*-
+import pytest
+from taxi_service.forms import SubscriberForm, MainOrderForm
+from app.models import SubscribeUsers
+
+
+@pytest.mark.parametrize(
```
Comment (SergeyKutsko) [inline app/tests/taxi_service_test/test_forms.py:7]: good

File: auto/settings.py [hunk 8]
```diff
@@ -110,9 +110,10 @@
 
 USE_TZ = True
 
-STATIC_URL = '/static/'
+STATIC_URL = '/prefix/'
```
Comment (SergeyKutsko) [inline auto/settings.py:113]: WTF back it to original it will brake production

File: fly.toml [hunk 9]
```diff
@@ -55,40 +53,9 @@ processes = []
     restart_limit = 0
     timeout = "2s"
 
-[[services]]
```
Comment (SergeyKutsko) [inline fly.toml:58]: WTF?

File: nginx/default.conf [hunk 10]
```diff
@@ -6,17 +6,17 @@ error_log /var/log/nginx/error.log;
 server {
     listen 80;
     listen [::]:80;
-
-    server_name ninja-taxi.kyiv.ua www.ninja-taxi.kyiv.ua localhost;
+    server_name  snowy-wood-371.fly.dev *.snowy-wood-371.fly.dev localhost ninja-taxi.kyiv.ua;
```
Comment (SergeyKutsko) [inline nginx/default.conf:9]: OMF back it to original

File: scripts/test_taxi_service_forms.py [hunk 11]
```diff
@@ -0,0 +1,39 @@
+from django.test import SimpleTestCase
+from taxi_service.forms import MainOrderForm, SubscriberForm
+
+
+class TestForms(SimpleTestCase):
```
Comment (SergeyKutsko) [inline scripts/test_taxi_service_forms.py:5]: good

File: app/models.py (hunk 4 above)
Comment (KurochkaR) [reply app/models.py:424]: add regex for numbers because w/o it numbers like 8095123456 go into db

**Comment by WARGDOOM** [comment]
1. Make a prompt under entering the phone number in the order
2. Make the font slightly smaller in the order field of the phone number and pickup location.
3. Hint and error far from subsribe form. Must be immediately under the form (as per order) 
subscribe без вводу даних
4. Make the font a little smaller in the subscribe field
5. The hint in subscribe is input without data in English, and input of incorrect data in Ukrainian. Make it in one language

**Comment by KurochkaR** [comment]
@WARGDOOM fixed

**Comment by WARGDOOM** [comment]
1. you need to remove the hint. It is not needed.
не потрібна підказка
2.Make the font slightly smaller than the prompt for the phone number order field and pickup location.
зробити чуть меьше шрифт
3.Make an even hint under entering the phone number in the order and pickup location.
зробити рівно!
4.The tip is far from the form. Must be immediately under the form (as per order)
потрібно підняти више як підказка в меню order
5. Make the font a little smaller in the subscribe prompt field (the same font as in the order prompt field)
зробити чуть меньше шрифт

**Comment by KurochkaR** [comment]
image

**Comment by KurochkaR** [comment]
how it works?(

**Comment by KurochkaR** [comment]
try again i remove prompt font was smaller in last commit but i dunno whyyou didn't see changes  @WARGDOOM

**Comment by WARGDOOM** [comment]
it works fine on Safari, but not on Chrome. why so?
screenshot from the Safari browser:
Снимок экрана 2023-03-01 в 13 11 31
Снимок экрана 2023-03-01 в 13 13 30

**Comment by KurochkaR** [comment]
maybe cache?

File: Dockerfile (hunk 1 above)
Comment (KurochkaR) [reply Dockerfile]: deleted

File: app/admin.py (hunk 2 above)
Comment (KurochkaR) [reply app/admin.py]: fixed

**Comment by WARGDOOM** [comment]
We still need to do html validation and javascript validation. order and subscribe

**Comment by KurochkaR** [comment]
what are the validation criteria?

**Comment by KurochkaR** [comment]
Remove reload, fix phone input @WARGDOOM @SpaceIgor

**Comment by KurochkaR** [comment]
@WARGDOOM check please

**Pull request merged**